tests/unit/test_utils.h

```diff
@@ -3,6 +3,17 @@
 #include <functional>
 #include "VecSim/vec_sim.h"
 
+VecSimIndex *CreateNewIndex(const HNSWParams &params);
+VecSimIndex *CreateNewIndex(const BFParams &params);
+
+size_t EstimateInitialSize(const HNSWParams &params);
+size_t EstimateInitialSize(const BFParams &params);
+
+size_t EstimateElementSize(const HNSWParams &params);
+size_t EstimateElementSize(const BFParams &params);
```

Comment:
K, it works
The whole problem started when I used the anonymous union in a templated function
so I thought it might cause a problem here also, but it's ok